guacamole/src/main/webapp/translations/en.json

```diff
@@ -355,6 +355,7 @@
         "FIELD_HEADER_SERVER_LAYOUT"   : "Keyboard layout:",
         "FIELD_HEADER_SFTP_DIRECTORY"   : "Default upload directory:",
         "FIELD_HEADER_SFTP_HOSTNAME"    : "Hostname:",
+        "FIELD_HEADER_SFTP_KEEPALIVE"   : "SFTP Keepalive Interval:",
```

Comment:
For consistency's sake, the field headers should be sentence case, not title case.